vignettes/MAGE.Rmd

```diff
@@ -131,30 +140,206 @@ CGM traces following the original description of MAGE calculations by
+|   Hall et al. (2018)    |     None      |   Adults (\>18)    | 16 36-69-001 |                5                 |
+|                         |               |                    | 16 36-69-026 |                6                 |
+| Tsalikian et al. (2005) |    Type 1     | Children (10 - 18) |      2       |                1                 |
+|                         |               |                    |      7       |                2                 |
+|                         |               |                    |      8       |                1                 |
+|                         |               |                    |      11      |                2                 |
+|                         |               |                    |      15      |                1                 |
+|                         |               |                    |      43      |                1                 |
+|                         |               |                    |      44      |                1                 |
+| Dubosson et al. (2005)  |    Type 1     |   Adults (\>18)    |      1       |                5                 |
+|                         |               |                    |      2       |                2                 |
+|   Broll et al. (2021)   |    Type 2     |   Adults (\>18)    |  Subject 1   |                1                 |
+|                         |               |                    |  Subject 2   |                5                 |
+|                         |               |                    |  Subject 3   |                1                 |
+|                         |               |                    |  Subject 4   |                5                 |
+|                         |               |                    |  Subject 5   |                6                 |
+
+*Existing MAGE Calculators*
+
+We benchmarked the proposed algorithm against a variety of open and
+closed-source tools. To the best of our knowledge, the list in Table 2
+comprises a relatively exhaustive list of existing packages to compute
+MAGE on CGM data as of May 15, 2024.
+
+**Table 2:** An exhaustive list (to the best of our knowledge) of
+existing packages and algorithms to compute MAGE as of May 15, 2024. We
+also specify the version and release date if we benchmark the proposed
+MAGE algorithm against it, or the reason we were unable to.
+
++-------------+-------------+-------------+-------------+-------------+
+| Tool or     | **I         | Be          | **Version   | Release /   |
+| Algorithm   | mplements   | nchmarked   | Used**      | Update Date |
+| Name        | MAGE**      | Against?    |             |             |
++:===========:+:===========:+:===========:+:===========:+:===========:+
+| iglu        | Yes         | Yes         | iglu v4.1   | Present     |
+| ::mage_ma ( |             |             |             |             |
```

Comment:
this has strange new line start

Reply:
I noticed these in the raw text, but when I attempt to make them one-line the table breaks

Reply:
Perhaps the version of knitr I'm using is incompatible